src/core/Akka.API.Tests/CoreAPISpec.ApproveCore.approved.txt

```diff
@@ -4902,4 +4903,4 @@ namespace Akka.Util.Reflection
     {
         public static System.Collections.Generic.IEnumerable<object> GetArguments(this System.Linq.Expressions.NewExpression newExpression) { }
     }
-}
\ No newline at end of file
+}
```

Comment:
Could you please double check that there isn't another line after the breached? I have pulled you branch and that was the last issue I had with the approval test even if the line doesn't show up here.

By the way you know that you can run the test locally ? :smile: 
It's locaded in the Akka.API.Tests projects